stan/math/mix/prob/laplace_latent_bernoulli_logit_rng.hpp

```diff
@@ -54,24 +58,28 @@ inline Eigen::VectorXd laplace_latent_tol_bernoulli_logit_rng(
  * return a multivariate normal random variate sampled
  * from the gaussian approximation of p(theta | y, phi),
  * where the likelihood is a Bernoulli with logit link.
+ * @tparam Mean type of the mean of the latent normal distribution
  * \laplace_common_template_args
  * @tparam RNG A valid boost rng type
  * @param[in] y Vector Vector of total number of trials with a positive outcome.
  * @param[in] n_samples Vector of number of trials.
+ * @param[in] mean the mean of the latent normal variable.
  * \laplace_common_args
  * \rng_arg
  * \msg_arg
  */
-template <typename CovarFun, typename CovarArgs, typename RNG>
+template <typename Mean, typename CovarFun, typename CovarArgs, typename RNG,
+          require_eigen_vector_t<Mean>* = nullptr>
```

Comment:
Should we delete this so that it is easier to make into a scalar later?
```suggestion
>
```